include/dlaf/communication/broadcast_panel.h

```diff
@@ -68,11 +70,26 @@ void broadcast(comm::IndexT_MPI rank_root, matrix::Panel<axis, T, D, storage>& p
   namespace ex = pika::execution::experimental;
   for (const auto& index : panel.iteratorLocal()) {
     if (rank == rank_root)
-      ex::start_detached(scheduleSendBcast(serial_comm(), panel.read(index)));
+      ex::start_detached(scheduleSendBcast(serial_comm.readwrite(), panel.read(index)));
     else
-      ex::start_detached(scheduleRecvBcast(serial_comm(), rank_root, panel.readwrite(index)));
+      ex::start_detached(scheduleRecvBcast(serial_comm.readwrite(), rank_root, panel.readwrite(index)));
+  }
+}
+
+// TODO: Move somewhere else?
```

Comment:
To do.

Reply:
Leaving where it is. It can't be a lambda with C++17 because of the template parameter.